weather_mv/loader_pipeline/ee.py

```diff
@@ -255,6 +258,12 @@ def add_parser_arguments(cls, subparser: argparse.ArgumentParser):
                                help='The expected latency per requests, in seconds. Default: 0.5')
         subparser.add_argument('--ee_max_concurrent', type=int, default=10,
                                help='Maximum concurrent api requests to EE allowed for your project. Default: 10')
+        subparser.add_argument('--band_names_mapping', type=str, default=None,
+                               help='A JSON file which contains the band names for the TIFF file.')
+        subparser.add_argument('--initialization_time', type=str, default=None,
```

Comment:
Can we rename `--initialization_time` to some other better name. 
`--initialization_time_parser`, WDYT ?
CC: @deepgabani8

Reply:
Lets keep it as `--initialization_time_regex`. And also change `--forecast_time` to `forecast_time_regex`.

Reply:
Resolved